bciers/apps/registration/app/components/operations/registration/FacilityInformationForm.tsx

```diff
@@ -98,8 +104,32 @@ const FacilityInformationForm = ({
     [setFormState],
   );
 
+  const redirect = () => {
+    // Using window.location.href instead of MutliStepBase router.push as there were rerender issues due to
+    // Facility Grid listerning to params change on this page which sometimes hijacked the route change
+    window.location.href = `${
+      window.location.origin
+    }/registration/register-an-operation/${operation}/${step + 1}`;
+  };
+
   const handleSubmit = useCallback(
```

Comment:
I think we should prevent the user from proceeding to the next page if they have opened the facility form but haven't entered any data into it, and don't have any existing facility for LFOs.

Reply:
Yes! This is how I meant for it to work, just messed up my conditionals :facepalm: